### ISSUE TITLE
Warning messages of unknown rules at the start of SonarQube

### ISSUE THREAD

**Issue by gvrf2911** [spotbugs/sonar-findbugs#357]
## Issue Description
Hello,

We have some warning messages at SonarQube startup about rules that cannot be activated because they are unknown.

These logs are as follows:

```
DEBUG web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Loaded deprecated profile definition FindBugs for language java | time=161ms
DEBUG web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Loaded deprecated profile definition FindBugs + FB-Contrib for language java | time=47ms
INFO  web[][o.s.s.r.RegisterRules] Disable rule findsecbugs:SPRING_ENTITY_LEAK
INFO  web[][o.s.s.r.RegisterRules] Disable rule findbugs:IRA_INEFFICIENT_REPLACEALL
INFO  web[][o.s.s.q.BuiltInQProfileRepositoryImpl] Load quality profiles
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'ENTITY_MASS_ASSIGNMENT'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'OVERLY_PERMISSIVE_FILE_PERMISSION'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'ENTITY_LEAK'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'GROOVY_SHELL'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'TEMPLATE_INJECTION_PEBBLE'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'SQL_INJECTION_VERTX'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'SAML_IGNORE_COMMENTS'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'IMPROPER_UNICODE'
DEBUG web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Loaded deprecated profile definition FindBugs Security Audit for language java | time=7ms
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'ENTITY_MASS_ASSIGNMENT'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'OVERLY_PERMISSIVE_FILE_PERMISSION'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'ENTITY_LEAK'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'GROOVY_SHELL'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'TEMPLATE_INJECTION_PEBBLE'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'SQL_INJECTION_VERTX'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'SAML_IGNORE_COMMENTS'
WARN  web[][o.s.s.q.BuiltInQProfileDefinitionsBridge] Unable to activate unknown rule : 'IMPROPER_UNICODE'
INFO  web[][o.s.s.q.RegisterQualityProfiles] Register quality
```
Do these warnings present a risk?

Otherwise, the plugin looks like to work fine with a Java demonstration project.

We have these warnings with an upgrade of the plugin or with starting from scratch with it.

We do not have these warnings with the version 3.11.1 of the plugin.

Thanks in advance.

## Environment

| Component          | Version |
| ------------------ | ------- |
| SonarQube          | 8.6-community   |
| Sonar-FindBugs     | 4.0.3   |
| Maven              | n/a |
| Gradle             | n/a   |
| Java               | 11  |

**Comment by gtoison** [comment]
@gvrf2911 thank you for reporting this issue, it should be solved in version 4.0.4 (and 4.0.5) of the plugin.

**Comment by gvrf2911** [comment]
Sorry for the delay in answer; I confirm that the issue has been solved in version 4.0.4 of the plugin.
Thank you.
